packages/expo-barcode-scanner/android/src/main/java/expo/modules/barcodescanner/BarCodeScannerViewManager.kt

```diff
@@ -0,0 +1,49 @@
+package expo.modules.barcodescanner
+
+import android.content.Context
+import expo.modules.core.ModuleRegistry
+import expo.modules.core.ModuleRegistryDelegate
+import expo.modules.core.ViewManager
+import expo.modules.core.interfaces.ExpoProp
+import expo.modules.interfaces.barcodescanner.BarCodeScannerSettings
+import java.util.*
+
+class BarCodeScannerViewManager(
```

Comment:
You can't remove `onCreate` method from `BarCodeScannerViewManager`. Something need to provide `ModuleRegistry` to `ModuleRegistryDelegate`.